### DISCUSSION TITLE
positioning  cursor horizontally

### DISCUSSION THREAD

**Discussion by ziat1988** [phase1geo/Minder#541] [General]
I have difficult when moving around a document. Right now the only way is use mouse scroll to move down or up. 
So for a large document, I have to zoom out then select the node I want.
Is there a way to use a "hand" to positioning anywhere in the document?

**Comment by phase1geo** [comment]
You can "pan" the canvas in a few different ways:

1. Hold down the middle button of the mouse and move it around.
2. Hold down the Alt key while moving the cursor around.
3. Hold down the Shift key while scrolling with the scroll wheel (this will scroll the map up/down instead of left/right).

**Comment by phase1geo** [comment]
FYI, panning the canvas (along with other mouse-related actions) is listed in the "Mouse Events" panel of the keyboard shortcuts cheatsheet (available from the header bar of the application).

**Comment by ziat1988** [comment]
Thank you for the quick answer. I use both of three way and the mouse middle is exact what I need.
It works perfectly.
Thanks for the great app.
